Examples/ManualExample/ManualExample/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,5 +1,15 @@
 {
   "images" : [
+    {
```

Comment:
What do these resource changes do?

Reply:
This is something Xcode must have done, and I'm not sure what triggered it.
Perhaps I should revert this part and see if anything breaks.

Reply:
FTR, if I undo the change, Xcode seems to redo it instantly.